message_definitions/v1.0/common.xml

```diff
@@ -1194,6 +1197,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="198" name="MAV_CMD_DO_SET_ROI_SYSID">
+        <description>Follow vehicle by system ID.</description>
+        <param index="1">sysid</param>
```

Comment:
This needs a label, ninValue and maxValue attributes

Reply:
Done.

Reply:
Done.